src/dataDisplay/personCoreMilestones/milestonePopoverContent.jsx

```diff
@@ -60,49 +60,46 @@ function MilestonesPopoverContent(props) {
 
     const labelColumnClasses = ClassNames(classes.milestoneColumn, classes.milestoneLabel);
 
-    const milestonesDatesJsx = milestonesDates.map((milestoneDate) => {
+    const milestonesDatesJsx = milestonesDates.map((milestoneDate, index) => {
         const {
             date,
             label,
         } = milestoneDate;
 
         return (
-            <Grid.Row
-                columns={2}
-                key={snakeCase(label)}
+            <Grid
+                className={index >= 1 && classes.dateGrid}
```

Comment:
Hmm, this might be better as:
```JavaScript
className={index >= 1 ? classes.dateGrid : undefined}
```
Maybe it's okay as-is (i.e. if index is 0 or 1, then `className` is `false`, and by JavaScript's whacky type rules, all is well)

Reply:
Fixed.

Reply:
Thanks!  Resolving.